vizro-core/src/vizro/models/_components/form/range_slider.py

```diff
@@ -83,14 +83,7 @@ def build(self):
 
         return html.Div(
             [
-                dcc.Store(
-                    f"{self.id}_callback_data",
-                    data={
-                        "id": self.id,
-                        "min": self.min,
-                        "max": self.max,
-                    },
-                ),
+                dcc.Store(f"{self.id}_callback_data", data={"id": self.id, "min": self.min, "max": self.max}),
```

Comment:
Can we add `id` key for the first attribute? So:
`dcc.Store(id=f"{self.id}_callback_data", data={"id": self.id, "min": self.min, "max": self.max}),`